docs/software/requesting-tokens.md

```diff
@@ -18,149 +17,150 @@ Before you can request the appropriate tokens, you must have the following:
+        For example, to request HTCondor `READ` and `WRITE` access, specify the following scopes:
 
-            For example, to request HTCondor `READ` and `WRITE` access, specify the following scopes:
-
-                This issuer supports the following scopes: openid profile email address phone offline_access wlcg iam wlcg.groups
-                Space delimited list of scopes or 'max' [openid profile offline_access]: wlcg offline_access compute.read compute.modify compute.cancel compute.create
+            This issuer supports the following scopes: openid profile email address phone offline_access wlcg iam wlcg.groups
+            Space delimited list of scopes or 'max' [openid profile offline_access]: wlcg offline_access compute.read compute.modify compute.cancel compute.create
     
-            Note that, prior to HTCondor 8.9.7, the server also needed `condor:/ALLOW` in all cases.
+        Note that, prior to HTCondor 8.9.7, the server also needed `condor:/ALLOW` in all cases.
 
-        1. When prompted, open <https://wlcg.cloud.cnaf.infn.it/device> in a browser, enter the code provided by `oidc-gen`,
+    1. When prompted, open <https://wlcg.cloud.cnaf.infn.it/device> in a browser, enter the code provided by `oidc-gen`,
        and click "Submit".
 
-        1. On the next page, verify the scopes and client profile name, and click "Authorize".
+    1. On the next page, verify the scopes and client profile name, and click "Authorize".
 
-        1. Enter a password to encrypt your local client profile.
+    1. Enter a password to encrypt your local client profile.
        You'll need to remember this if you want to re-use this profile in subsequent sessions.
 
-1. Request a token using the client profile that you used above with `oidc-gen`:
+### Requesting access tokens
 
-    !!! note
-        You must first [register a new profile](#using-a-container)
-	
-    1. Request token:
+!!! note
+    You must first [register a new profile](#registering-an-oidc-profile)
 
-            :::console
-            docker exec -it my-agent oidc-token --aud="<SERVER AUDIENCE>" <CLIENT PROFILE>
+1. Request a token using the client profile that you used with `oidc-gen`:
+	
+        :::console
+        docker exec -it my-agent oidc-token --aud="<SERVER AUDIENCE>" <CLIENT PROFILE>
 
 
-    1. For tokens used against an HTCondor-CE, set `<SERVER AUDIENCE>` to  
+For tokens used against an HTCondor-CE, set `<SERVER AUDIENCE>` to  
     `<CE FQDN>:<CE PORT>`.
 
-        1. Copy the output of `oidc-token` into a file on the host where you need SciToken authentication, e.g. an HTCondor or
+  1. Copy the output of `oidc-token` into a file on the host where you need SciToken authentication, e.g. an HTCondor or
    XRootD client.
 
-1. Reload a profile:
+### Reloading an OIDC profile
 
-    1. Required if you restart the running container:
+!!! note
+    Required after restarting the running container.
```

Comment:
This also requires an existing profile so we should link to the relevant profile registration like the "Requesting access tokens" note